runtime/onert/core/src/compiler/train/pass/BiasInsertionPass.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_COMPILER_TRAIN_PASS_BIAS_INSERTION_PASS_H__
+#define __ONERT_COMPILER_TRAIN_PASS_BIAS_INSERTION_PASS_H__
+
+#include "../../pass/Pass.h"
+#include "ir/OperationVisitor.h"
+
+namespace onert
+{
+namespace compiler
+{
+namespace train
+{
+namespace pass
+{
+
+class BiasInsertionPass final : public compiler::pass::Pass, public ir::OperationVisitor
+{
+public:
+  BiasInsertionPass(ir::Graph &graph) : compiler::pass::Pass{graph} {}
+
+public:
+  std::string id() final { return "BiasInsertionPass"; }
+  void run() final;
+
+public:
+  void visit(const ir::operation::FullyConnected &node) override;
```

Comment:
Should we have to insert bias into each node? 
I'd like to review this PR, But I feel hard to understand the background :smiling_face_with_tear:

Reply:
Sometimes TFLite converter may generate a inference model that does not have bias input(i.e. optional input) when the bias data has all zero values. It's reasonable in inference because there is no reason to add zero values. But, in training, bias is necessary because bias needs to be trained and updated.
As a result, we need to insert bias input if a model to be trained does have bias input.

Reply:
I found out tensorflow cannot use bias for [`Dense`](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense) layer by using `use_bias` parameter.
```python
tf.keras.layers.Dense(
    units,
    activation=None,
    use_bias=True,
    ...
)
```
It means a converted model with optional bias such as the below model might be trained without bias intentionally. If so, the training results is the same as the result of training without bias.
![image](https://github.com/user-attachments/assets/5caaba46-07d2-4786-8791-cb2780d5747d)
When converted models have optional bias, how can we figure out that the bias was intentionally removed?